.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -32,26 +32,26 @@ body:
     attributes:
       label: Files Version
       description: Which version of Files are you using? The version number can be found by going to Files' Settings > About
-      placeholder: "e.g. Version: 2.0.34.0 OS Version: 10.0.22000.282"
+      placeholder: "e.g. Version: 2.4.0.0 OS Version: 10.0.22621.900"
```

Comment:
Let's keep the placeholder for now, it's a good way to spot when the template wasn't filled out.